Scoop/View/EditProfileViewController.swift

```diff
@@ -0,0 +1,256 @@
+        profileImageView.contentMode = .scaleAspectFill
+        profileImageView.clipsToBounds = true
+        profileImageView.layer.cornerRadius = 60
+        profileImageView.layer.borderColor = UIColor.scoopGreen.cgColor
+        profileImageView.layer.borderWidth = 3
+        view.addSubview(profileImageView)
+        
+        profileImageView.snp.makeConstraints { make in
+            make.top.equalTo(view.safeAreaLayoutGuide).offset(35)
+            make.size.equalTo(120)
+            make.centerX.equalToSuperview()
+        }
+    }
+    
+    private func setupUploadPhotoButton() {
+        uploadPhotoButton.setImage(UIImage(named: "profilebutton"), for: .normal)
+        uploadPhotoButton.backgroundColor = .red
+        uploadPhotoButton.contentMode = .scaleAspectFill
+        uploadPhotoButton.clipsToBounds = true
+        uploadPhotoButton.layer.cornerRadius = 13
+        view.addSubview(uploadPhotoButton)
+        
+        uploadPhotoButton.snp.makeConstraints { make in
+            make.top.equalTo(profileImageView).offset(96)
+            make.size.equalTo(26)
+            make.leading.equalTo(profileImageView).offset(85)
+            make.trailing.equalTo(profileImageView).offset(-9)
+        }
+    }
+
+    private func setupStackView() {
+        let leadingTrailingInset = 32
+        let textFieldHeight = 56
+        
+        stackView.axis = .vertical
+        stackView.distribution = .fill
+        stackView.alignment = .leading
+        stackView.spacing = 24
+        view.addSubview(stackView)
+        
+        stackView.snp.makeConstraints { make in
+            make.leading.trailing.equalToSuperview().inset(leadingTrailingInset)
+            make.top.equalTo(profileImageView.snp.bottom).offset(25)
+        }
+        
+        nameTextField.textColor = UIColor.black
+        nameTextField.borderStyle = .roundedRect
+        stackView.addArrangedSubview(nameTextField)
+        
+        nameTextField.snp.makeConstraints { make in
+            make.leading.trailing.equalToSuperview()
+            make.height.equalTo(textFieldHeight)
+        }
+        
+        pronounsTextField.textColor = UIColor.black
+        pronounsTextField.borderStyle = .roundedRect
+        stackView.addArrangedSubview(pronounsTextField)
+        
+        pronounsTextField.snp.makeConstraints { make in
+            make.leading.trailing.equalToSuperview()
+            make.height.equalTo(textFieldHeight)
+        }
+        
+        hometownTextField.textColor = UIColor.black
+        hometownTextField.borderStyle = .roundedRect
+        stackView.addArrangedSubview(hometownTextField)
+        
+        hometownTextField.snp.makeConstraints { make in
+            make.leading.trailing.equalToSuperview()
+            make.height.equalTo(textFieldHeight)
+        }
+        
+        classTextField.textColor = UIColor.black
+        classTextField.borderStyle = .roundedRect
+        stackView.addArrangedSubview(classTextField)
```

Comment:
The font for these text views look a little off from Scooped's regular fonts. Idk if design has handed off the Figma designs yet but the placeholder text doesn't like the ones we have in the text boxes in onboarding